docs/content/docs/getting-started.md

```diff
@@ -197,25 +197,35 @@ layout: MySuperLayout
 featuredImage: someImageUrl.png # ← Add your image here
 ---
 
-My super **content** !
+My super **content**!
 ```
 
 ---
 
 ⚠️ It is possible to add any information in the front-matter, and you might be
-able to do almost anything you can think of, even images gallery!
+able to do almost anything you can think of, even add an images gallery!
 
 ---
 
-The front-matter data are send like page body in your React components.
+The front-matter data is sent with `props` in your React components, and is
+available under `props.head`.
 This allow you to manipulate it normally using JavaScript.
 
 Supposing you want to use your ``featuredImage`` in a list of pages or posts (``PagesList`` in the default theme)
 you will have to change your ``PagePreview`` to make use of that since ``PagesList`` just render a series of ``PagePreview`` components.
 
 Changing the ``PagePreview`` component will change how the "latest posts" are rendered on your homepage.
 
-```js
+---
+
+⚠️ The `Homepage` layout filters the latest posts by `Post` layout. If you don't
+see your new page in this list, either change your new page to have the `Post`
+layout, or edit the homepage collections filter to include your custom layout
+by checking out the [How to use Collections](https://phenomic.io/docs/usage/collections/) page.
```

Comment:
Maybe this is too verbose, but I thought it was hard to follow the example new page we were setting up after we had created a custom layout. I didn't understand why my new page wasn't showing up in the list of posts on the homepage until I saw the collections filter.